src/client/src/components/ManageAccounts.jsx

```diff
@@ -0,0 +1,164 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { Context } from "../UserSession";
+import { useContext } from "react";
+import swal from "sweetalert";
+import { useTranslation } from "react-i18next";
+
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Paper,
+  Button,
+} from "@mui/material";
+import { Link } from "react-router-dom";
+import "../css/ManageAccounts.css";
+
+const ManageAccouts = () => {
+  const [accounts, setAccounts] = useState([]);
+  const { t, i18n } = useTranslation();
+  //Global loginState
+  const [login, setLogin] = useContext(Context);
+
+  //Get id of logged in user
+  const userID = sessionStorage.getItem("userID");
+
+  useEffect(() => {
+    if (userID) {
+      fetchSession();
+    }
+  }, []);
+
+  //Having the loginState persist on all pages
+  const fetchSession = async () => {
+    try {
+      if (userID) {
+        setLogin({
+          isLoggedIn: true,
+        });
+      }
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  useEffect(() => {
+    fetchAccounts();
+  }, []);
+
+  const fetchAccounts = async () => {
+    try {
+      const response = await axios.get(`http://localhost:9000/admin/users`);
+      console.log(userID);
+      setAccounts(response.data);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const ban = async (accountuserID) => {
+    axios
+      .post(`http://localhost:9000/admin/ban/${accountuserID}`, {
+        isBan: true,
+      })
+      .then((response) => {
+        console.log(response.data);
+        swal(t("Saved!"), t("Successfully updated the user"), "success", {
+          button: false,
+          timer: 2000,
+        });
+      })
+      .catch((error) => {
+        console.log(error);
+        swal(t("Failed!"), t("Cannot update the user"), "error", {
+          button: false,
+          timer: 2000,
+        });
+      });
+  };
+
+  const unban = async (accountuserID) => {
+    axios
+      .post(`http://localhost:9000/admin/unban/${accountuserID}`, {
+        isBan: true,
```

Comment:
Can also remove the request body since we isBan is set directly to false in the controller

Reply:
remove the isban from the axios body call